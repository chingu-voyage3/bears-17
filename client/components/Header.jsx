import React from 'react';

import Navigation from 'Components/Navigation';
import Logo from 'Components/Logo';

const Header = () => (
  <header className="site-header">
    <div className="wrapper wrapper--flex">
      <Logo />
      <Navigation />
    </div>
  </header>
);

export default Header;
