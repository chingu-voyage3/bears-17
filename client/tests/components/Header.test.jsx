import React from 'react';
import renderer from 'react-test-renderer';

import Header from 'Components/Header';

describe('Header', () => {
  it('should match the snapshot', () => {
    const header = renderer.create(<Header />);

    expect(header.toJSON()).toMatchSnapshot();
  });
});
