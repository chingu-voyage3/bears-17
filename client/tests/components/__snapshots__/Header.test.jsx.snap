// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<header
  className="site-header"
>
  <div
    className="wrapper wrapper--flex"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="/"
      >
        <img
          alt="Our app's name"
          src="test-file"
        />
      </a>
    </div>
    <nav
      aria-label="Site links"
    >
      <ul
        className="main-nav"
      >
        <li
          className="main-nav__item"
        >
          <a
            className="main-nav__link"
            href="/"
          >
            Home
          </a>
        </li>
        <li
          className="main-nav__item"
        >
          <a
            className="main-nav__link"
            href="/about"
          >
            About
          </a>
        </li>
        <li
          className="main-nav__item"
        >
          <a
            className="main-nav__link"
            href="/login"
          >
            Login
          </a>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
